### PR TITLE
Add documentation for version in templates

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Adds documentation for version in templates.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/78484
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/templating.markdown [hunk 1]
```diff
@@ -548,6 +548,20 @@ Example using `is_defined` to parse a JSON payload:
...
+- `version()` Returns a [AwesomeVersion object](https://github.com/ludeeus/awesomeversion) for the value given inside the brackets.
+  - This is also available as a filter (`| version`
+
+Examples:
+
+{% raw %}
+
+- `{{ version("2099.9.9") > "2000.0.0" }}` Will return `True`
+- `{{ version("2099.9.9") < "2099.10" }}` Will return `True`
+- `{{ "2099.9.9" | version < "2000.10" }}` Will return `True`
```
Comment (ludeeus) [inline source/_docs/configuration/templating.markdown]:
```suggestion
- `{{ "2099.9.9" | version < "2099.10" }}` Will return `True`
```

File: source/_docs/configuration/templating.markdown [hunk 2]
```diff
@@ -548,6 +548,20 @@ Example using `is_defined` to parse a JSON payload:
 
 This will throw an error `UndefinedError: 'value_json' is undefined` if the JSON payload has no `val` attribute.
 
+### Version
+
+- `version()` Returns a [AwesomeVersion object](https://github.com/ludeeus/awesomeversion) for the value given inside the brackets.
+  - This is also available as a filter (`| version`
```
Comment (frenck) [inline source/_docs/configuration/templating.markdown]:
```suggestion
  - This is also available as a filter (`| version`).
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_docs/configuration/templating.markdown [hunk 3]
```diff
@@ -548,6 +548,20 @@ Example using `is_defined` to parse a JSON payload:
...
+- `version()` Returns a [AwesomeVersion object](https://github.com/ludeeus/awesomeversion) for the value given inside the brackets.
+  - This is also available as a filter (`| version`).
+
+Examples:
+
+{% raw %}
+
+- `{{ version("2099.9.9") > "2000.0.0" }}` Will return `True`
+- `{{ version("2099.9.9") < "2099.10" }}` Will return `True`
+- `{{ "2099.9.9" | version < "2099.10" }}` Will return `True`
```
Comment (ludeeus) [inline source/_docs/configuration/templating.markdown:562]:
```suggestion
- `{{ "2099.9.9" | version < "2099.10" }}` Will return `True`
- `{{ (version("2099.9.9") - "2100.9.10").major }}` Will return `True`
- `{{ (version("2099.9.9") - "2099.10.9").minor }}` Will return `True`
- `{{ (version("2099.9.9") - "2099.9.10").patch }}` Will return `True`
```

**Review by frenck — APPROVED** [review]

**Pull request merged**